src/vmm/src/persist.rs

```diff
@@ -37,8 +39,7 @@ use vm_memory::GuestMemoryMmap;
 const FC_V0_23_SNAP_VERSION: u16 = 1;
 #[cfg(target_arch = "x86_64")]
 const FC_V0_23_IRQ_NUMBER: u32 = 16;
-#[cfg(target_arch = "aarch64")]
-const FC_V0_23_IRQ_NUMBER: u32 = 128;
+#[cfg(target_arch = "x86_64")]
 const FC_V0_23_MAX_DEVICES: u32 = FC_V0_23_IRQ_NUMBER - IRQ_BASE;
```

Comment:
At that point, we thought it would be better show where that limitation is coming from, instead of just hardcoding the value. However, it is true that changing `IRQ_BASE` in the future would affect the logic. I will change this.